pyproject.toml

```diff
@@ -36,7 +36,7 @@ packages = [
 ]
```

Comment:
I think line `25` needs to be removed too? 
```
    "Programming Language :: Python :: 3.6",
```

Reply:
Thanks, @hughhm. 
This line is now removed, also add one entry for 3.9.